docs/organizations/managing-security-and-risk.md

```diff
@@ -104,6 +104,36 @@ The following table defines item priorities and days to fix the underlying Codac
 | Medium                  | Medium        | 90            |
 | Minor                   | Low           | 120           |
 
+## Access management
```

Comment:
I like how you're considering adding this new section directly under the Managing security and risk feature. It has the advantage of keeping things more self-contained, at least for now. :+1:

Reply:
Correct, and as you're saying...at least for now. We now have three PRs targeting the same branch (which is itself a PR...), some with quite extensive changes. I'm keeping things as isolated as possible, at least on these lesser updates.